paid/ui/assets/extensions/paid-online-extensions.js

```diff
@@ -0,0 +1,12 @@
+/*
+ * Online extensions specific to pro tier
+ */
+
+(function() {
+  'use strict';
+
+  window.OPENSHIFT_CONSTANTS.QUOTA_NOTIFICATION_MESSAGE = {
+    "limits.memory": "You can <a href=\""+window.OPENSHIFT_EXTENSION_PROPERTIES.account_url+"\">upgrade your plan</a> if you need more resources",
+    "limits.cpu": "You can <a href=\""+window.OPENSHIFT_EXTENSION_PROPERTIES.account_url+"\">upgrade your plan</a> if you need more resources"
```

Comment:
Some suggested edits:

- `<a>Add additional resources</a> to increase the quota.`
- `<a>Manage your subscription</a> to add additional resources.`
- `<a>Manage your subscription</a> to increase the quota.`